### DISCUSSION TITLE
mpv的watch-later怎么设置能自动恢复上次外挂的字幕？

### DISCUSSION THREAD

**Discussion by zhongfly** [hooke007/mpv_PlayKit#47] [Q&A 问答]
如题，用sid只能恢复内置的字幕……
(假如字幕与视频文件名不同，不会被自动加载）

虽然这个需求有些不必要……

**Comment by hooke007** [comment]
外挂字幕应该实际是这个选项传递进去的 `--sub-files`
但是它不被保存

**Reply by zhongfly** [reply]
可能只支持保存这些选项 https://github.com/mpv-player/mpv/commit/1d1d1fbff9648e9adf7acf571514abf618ffc40f#diff-b4539fc01c0e9fa72518b27234388d43e6d33759864ff4d141ba85f9bace3178

**Reply by hooke007** [reply]
去掉可能 https://hooke007.github.io/mpv-lazy/[04]_%E5%B8%B8%E8%A7%81FAQ%EF%BC%88%E8%B8%A9%E5%9D%91%E5%BD%95%EF%BC%89.html#q-%E9%80%80%E5%87%BA%E6%97%B6%E5%8F%AF%E4%BF%9D%E5%AD%98%E7%9A%84%E5%85%B6%E5%AE%83%E9%80%89%E9%A1%B9

**Comment by dyphire** [comment]
目前应该是无法实现的。关于外挂相关我推荐使用这几个lua脚本搭配来改善mpv糟糕的外挂逻辑：
外挂路径检测增强--[fuzzydir.lua](https://github.com/dyphire/MPV-own/blob/master/scripts/fuzzydir.lua)，指定音轨优先级/黑白名单--[trackselect.lua](https://github.com/po5/trackselect)，指定字幕优先级/黑白名单--[sub-select.lua](https://github.com/CogentRedTester/mpv-sub-select)。

功能介绍及示例：[scripts](https://github.com/dyphire/MPV-own/blob/master/scripts/README.md)
